DynamicDependenciesSample/DynamicDependencies/DirectX/D3D9ExSample.vcxproj

```diff
@@ -36,7 +40,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
-    <PlatformToolset>v142</PlatformToolset>
+    <PlatformToolset>v143</PlatformToolset>
     <CharacterSet>Unicode</CharacterSet>
```

Comment:
while we're in here, we should consolidate common settings like this to make future updates simpler.  and we should make this a default to allow overriding (upgrading)